src/bundle/Resources/public/css/modules/table-data.css

```diff
@@ -32,7 +32,7 @@
     display: inline-block;
 }
 
-.ez-table-data td:last-of-type {
+.ez-table-data-row-tool td:last-of-type {
```

Comment:
@inakijv added new styles for tables to the guidelines today. With those tables, I think we cover most of the tables in the administration interface.
https://ezsystems.github.io/ui-guidelines/#components-tables

`ez-table-data-row-tool` is more generic than `ez-table-checkboxes`. So +1
But we will have to find names and a simple way to manage table styles.